test/github_tests/test_github.py

```diff
@@ -370,3 +370,50 @@ def test_request_reviews_without_repo(self, mock_get_reviews, mock_github_patch)
         mock_github_instance.get_user.assert_called_with("dummy_user")
         mock_github_patch.assert_called_with("dummy_token")
         self.assertEqual(["1"], response)
+
+    @patch(PATH + "GithubService.check_request_state")
+    @patch(PATH + "GithubService.get_last_comment")
+    def test_get_reviews_with_draft_status(
+        self,
+        mock_last_comment,
+        mock_check_request_state,
+    ):
+        """
+        Tests get_reviews() function.
+
+        Where getting the pull requests returns single PR flagged as draft.
+        """
+        # Set up mock return values and side effects
+        mock_uname = MagicMock()
+        mock_repo = MagicMock()
+        mock_pr = mock_github.MockPull()
+        mock_pr.draft = True
+        mock_repo.get_pulls.return_value = [mock_pr]
+        mock_uname.get_repo.return_value = mock_repo
+        mock_check_request_state.return_value = True
+
+        for title, expected_title in [
+            ("dummy_title", "WIP: dummy_title"),
+            ("WIP: dummy_title", "WIP: dummy_title"),
+            ("Draft: dummy_title", "Draft: dummy_title"),
```

Comment:
Might also consider validating all-lower case.
The regex seems to allow mixed case vi `re.IGNORECASE`.

Reply:
Might also try testing the `[WIP] `  square-brace variation.

Reply:
I have not changed this, only reformatted the code with black.

Reply:
Ah, actually, my mistake, that is the oldest commit in this PR. :facepalm: